### PR TITLE
fix: Revert changes to adaptive card templates to support PVA

### LINKED ISSUE #7394: PVA: Valid adaptive card json prevents publishing to PVA
## Repro
* Create a non-PVA bot with adaptive card, test that it works.
* Create a PVA bot, switch to advanced authoring with composer
* Reproduce the adaptive card in PVA bot
* Publish

## Error
```
"severity": "Error",
"source": "MoodCheckerDialog.en-us.lg",
"message": "Error occurred when parsing expression 'json({\r\n \"$schema\": \"http://adaptivecards.io/schemas/adaptive-card.json\",\r\n \"version\": \"1.2\",\r\n \"type\": \"AdaptiveCard\",\r\n \"body\": [\r\n {\r\n \"type\": \"Container\",\r\n \"items\": [\r\n {\r\n \"type\": \"ColumnSet\",\r\n \"columns\": [\r\n {\r\n \"type\": \"Column\",\r\n \"width\": \"auto\",\r\n \"items\": [\r\n {\r\n \"type\": \"Image\",\r\n \"url\": \"https://metatour.azurewebsites.net/img/happy.png\",\r\n \"size\": \"small\"\r\n '. missing '}' at '<EOF>'"
}
---
```

## Sample JSON
```
# SendActivity_kUwtny()
[Activity
    Attachments = ${SendActivity_kUwtny_attachment_fpA8bl()}
]

# SendActivity_kUwtny_attachment_fpA8bl()
> To learn more Adaptive Cards format, read the documentation at
> https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
- \```${json({
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
        "type": "Container",
        "items": [
          {
            "type": "ColumnSet",
            "columns": [
              {
                "type": "Column",
                "width": "auto",
                "items": [
                  {
                    "type": "Image",
                    "url": "https://metatour.azurewebsites.net/img/sad.png",
                    "size": "small"
                  }
                ]
              },
              {
                "type": "Column",
                "width": "stretch",
                "items": [
                  {
                    "type": "TextBlock",
                    "text": "I'm sorry to hear that. Feel better soon.",
                    "weight": "bolder",
                    "wrap": true,
                    "BlockElementHeight" : "stretch"
                  }
                ]
              }
            ]
          }
        ]
    }
  ]
})}\```
(backslash inserted for rendering in issue comment)
```

### PR DESCRIPTION
**Pull request by tdurnford**
## Description
 Revert template changes in PR #7593 to support PVA

## Task Item
Closes #7394

### CODE REVIEW

File: Composer/packages/lib/code-editor/src/lg/constants.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export type LgCardTemplateType = typeof lgCardAttachmentTemplates[number];
 export const cardTemplates: Record<LgCardTemplateType, string> = {
   adaptive: `> To learn more Adaptive Cards format, read the documentation at
 > https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
-- \${{
+- \`\`\`\${json({
```
Comment (cwhitten) [inline Composer/packages/lib/code-editor/src/lg/constants.ts:25]: we don't want the use of `json` here, the following should work:
```
- ```{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "text": "default text",
      "weight": "bolder",
      "isSubtle": false
    }
  ]
}```
```

**Review by cwhitten — CHANGES_REQUESTED** [review]

File: Composer/packages/lib/code-editor/src/lg/constants.ts (hunk 1 above)
Comment (tdurnford) [reply Composer/packages/lib/code-editor/src/lg/constants.ts:25]: Oh, I see. So we still want the Activity template to use`${json(template_name()}`, but we don't want to use json in the adaptive card template string.

Comment (cwhitten) [reply Composer/packages/lib/code-editor/src/lg/constants.ts:25]: Yes, but the following also works in PVA:

```
# SendActivity_yhVIg9()
[Activity
    Attachments = ${SendActivity_yhVIg9_attachment_h83mGt()}
]

# SendActivity_yhVIg9_attachment_h83mGt()
> To learn more Adaptive Cards format, read the documentation at
> https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
- ```{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "text": "default text",
      "weight": "bolder",
      "isSubtle": false
    }
  ]
}```
```

Comment (cwhitten) [reply Composer/packages/lib/code-editor/src/lg/constants.ts]: Yes TJ exactly

Comment (tdurnford) [reply Composer/packages/lib/code-editor/src/lg/constants.ts]: When I exclude the `json()` all together, it doesn't work in the Web Chat panel.
```
# SendActivity_yhVIg9()
[Activity
    Attachments = ${SendActivity_yhVIg9_attachment_h83mGt()}
]

# SendActivity_yhVIg9_attachment_h83mGt()
> To learn more Adaptive Cards format, read the documentation at
> https://docs.microsoft.com/en-us/adaptive-cards/getting-started/bots
- ```{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2",
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "TextBlock",
      "text": "default text",
      "weight": "bolder",
      "isSubtle": false
    }
  ]
}```
```

Comment (cwhitten) [reply Composer/packages/lib/code-editor/src/lg/constants.ts]: That's right I tested the same thing myself (edited my previous message)

**Review by cwhitten — APPROVED** [review]

**Pull request merged**